cmd/appgw-ingress/main.go

```diff
@@ -133,3 +145,15 @@ func getKubeClientConfig() *rest.Config {
 
 	return config
 }
+
+func getEventRecorder(kubeClient kubernetes.Interface) (record.EventRecorder, error) {
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartLogging(glog.Infof)
+	eventBroadcaster.StartRecordingToSink(
+		&typedcorev1.EventSinkImpl{
+			Interface: kubeClient.CoreV1().Events("")})
+	recorder := eventBroadcaster.NewRecorder(
+		scheme.Scheme,
+		v1.EventSource{Component: "controlplane"})
```

Comment:
1. It'd be nice to put `"controlplane"` in constant somewhere. Also, shouldn't this string be something with higher specificity - perhaps `appgw.ingress.kubernetes.io` ?
2. Should we also add `Host` here in the `EventSource` struct to get a complete picture of where the event is coming from?